README.md

```diff
@@ -1,44 +1,51 @@
+<p align="center"><img width=50% src="docs/source/images/compose.png" alt="Compose" /></p>
```

Comment:
To render on PyPI, we'll need to switch back to absolute paths for the images before merging to main.

```
https://raw.githubusercontent.com/FeatureLabs/compose/main/docs/source/images/compose.png
```